### ISSUE TITLE
"DIAGNOSE" doesn't work as intended in 2081?

### ISSUE THREAD

**Issue by eriktorbjorn** [the-infocom-files/amfv#53]
This is part of ```V-DIAGNOSE``` that is specific to the 2081 simulation:

```zil
	       (<EQUAL? ,SYEAR 2081>
		<SCORE 40> ;"2 points"
		<TELL
"You are weak from hunger, a dangerous condition at your age. Your last meal
was... you can't even remember. You remember picking at the remains of a
squirrel, but that might have been weeks ago. Your memories are jumbled.">
		<COND (<FSET? ,CEMETERY ,ONBIT>
		       <TELL
" In addition, your jaw is throbbing from that attack in the cemetery.">)>
		<CRLF>)
```

But it doesn't seem to work quite as intended:

```
>DIAGNOSE
You are weak from hunger, a dangerous condition at your age. Your last meal
was... you can't even remember. You remember picking at the remains of a
squirrel, but that might have been weeks ago. Your memories are jumbled.

>NORTH
Cemetery
Once a lovely old cemetery, the land is now overgrown and the headstones toppled
and vandalized. Snarled bushes and trees provide countless hiding places for
thieves and cutthroats.

With a rustle of leaves, someone leaps out of concealment. You barely catch a
glimpse of a grizzled man wearing a torn imi-denim outfit before a fist cracks
you across the face, and you pass out as you begin rolling down the hillside.
Sometime later, you come to...

Main & Wicker
The street runs from west to east, where a bridge is visible. A crude sign is
posted just west of here. Another road leads south into a different section of
town, where smoke billows upward from scattered fires. Buildings all along the
street look typically looted and deserted; a grocery store, its large plate
glass window shattered, is on the southwest corner. On the southeast corner is
the burnt-out shell of a building, and an overgrown cemetery lies to the north.
As usual, all the telephone poles have been chopped down for firewood.

>DIAGNOSE
Your head throbs from the blow you took during the mugging.
```

That's because ```CEMETERY-F``` sets ```MUGGED```, which takes precedence:

```zil
	       (<AND <EQUAL? .RARG ,M-END>
		     <EQUAL? ,SYEAR 2081>>
		<TELL CR
"With a rustle of leaves, someone leaps out of concealment. You barely
catch a glimpse of a grizzled man wearing a torn imi-denim outfit before
a fist cracks you across the face, and you pass out as you begin rolling
down the hillside. ">
		<COND (<FSET? ,CEMETERY ,ONBIT>
		       <JIGS-UP "This time, you fail to recover.">)
		      (T
		       <TELL "Sometime later, you come to..." CR CR>
		       <FSET ,CEMETERY ,ONBIT>
		       <SETG MUGGED T>
		       <ROB ,PLAYER ,LOCAL-GLOBALS>
		       <INCREMENT-TIME 23>
		       <GOTO ,MAIN-AND-WICKER>)>)>>
```

But does it even need to set that? As far as I can tell, ```MUGGED``` is only tested by ```V-DIAGNOSE```, and it's not needed for the desired message here.
